securebanking-openbanking-uk-rcs-sample/docker/config/securebanking-openbanking-uk-rcs-docker.yml

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright © 2020 ForgeRock AS (obst@forgerock.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+### Spring Configuration for securebanking-openbanking-uk-rcs running in Docker
+
+dns:
+  hosts:
+    root: localhost
```

Comment:
We need to have a discussion about this, but I've followed the same pattern as OB toolkit for now.